assets/js/googlesitekit/datastore/user/utils.test.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * core/user Data store: utils tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+
+import {
+	getDateString,
+	getPreviousDate,
+	getPreviousWeekDate,
+	isValidDateInstance,
+} from './utils';
+
+describe( 'core/user utils', () => {
+	const INVALID_DATE_INSTANCE_ERROR = 'Date param must construct to a valid date instance or be a valid date instance itself.';
+	describe( 'isValidDateInstance', () => {
+		it( 'should return true for valid date instance (no constructor params)', () => {
+			expect( isValidDateInstance( new Date() ) ).toEqual( true );
+		} );
+		it( 'should return true for valid date instance (valid constructor params)', () => {
```

Comment:
Please add blank lines between `it` calls for consistency with other tests and readability.
```suggestion
		} );

		it( 'should return true for valid date instance (valid constructor params)', () => {
```